presto-main/src/main/java/io/prestosql/execution/QueryPreparer.java

```diff
@@ -15,6 +15,7 @@
 
 import com.google.common.collect.ImmutableList;
```

Comment:
Please shorten your commit message title. It should be max 70 chars.

Reply:
ok

Reply:
Ok